openedx/core/djangoapps/schedules/management/commands/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 class SendEmailBaseCommand(PrefixedDebugLoggerMixin, BaseCommand):
     async_send_task = None  # define in subclass
-    offsets = ()  # define in subclass
+    offsets = xrange(-7, -77, -7)
```

Comment:
It's not obvious at first what this `offsets` parameter is doing... can we add a comment with a bit more explanation? e.g. "An iterable of day offsets (e.g. -7, -14, -21, -28, ...) that defines the days for which emails are sent out, relative to the 'date' parameter"

Reply:
@DevanR  ^